Terraform/AWS/task5/instance.tf

```diff
@@ -0,0 +1,69 @@
+resource "aws_instance" "public" {
+  ami           = data.aws_ami.ubuntu.id
+  instance_type = "t3.micro"
+  subnet_id     = aws_subnet.private[0].id
+  associate_public_ip_address = true
+  availability_zone = data.aws_availability_zones.available.names[0]
+  key_name = "ssh key"
+  tags = {
+    Name = "${var.env_code}-public"
+  }
+}
+resource "aws_instance" "private" {

```

Comment:
created security group but didn't associate it with the instance

Reply:
I have this question, I tried to drop the first commit "assign and add new line...." but it is still here